src/components/cards/MoreLikeThis.js

```diff
@@ -95,7 +107,12 @@ export default function MoreLikeThis({ course }) {
         </div>
         <div className='flex justify-between mt-10'>
           <div className='flex gap-2'>
-            <ViewBtn id={data.hits[0].meta.id} />
+            <ViewBtn onClick={() => {
+              const domain = (new URL(window.location));
+              const objectId = `${domain.origin}/course/${data.hits[0].meta.id}`;
+              xAPISendStatement(objectId);
+            }}
+            id={data.hits[0].meta.id} />
```

Comment:
Building off the above question. This seems to be some generic code in terms of use. We may want to consider moving this into the button its intended for.

Reply:
Addressed.